### ISSUE TITLE
Normalize case for package names

### ISSUE THREAD

**Issue by saulshanabrook** [elegion/djangodash2013#1]
For instance [saulshanabrook/django-canadanewyork](http://djangodash13.glyuck.com/projects/python/django-canadanewyork) lists `Django` and is reccommended `django`. Maybe store all packages as lower case?

**Comment by glyuck** [comment]
Good catch! Looks like pkgtools.pypi can handle it, and I broke it in one of commits trying to speed up repos analyzing. Will review this commit and maybe revert it.
https://github.com/elegion/djangodash2013/blob/master/wtl/wtparser/parsers/requirements_txt.py#L61

``` python
# TODO: get rid of fast=true (need to normalize package name)
json = PyPIJson(package_name, fast=True).retrieve(_request)
```
